contrib/completion/bash/docker

```diff
@@ -5482,10 +5488,14 @@ _docker() {
 	)
 
 	local known_plugin_commands=()
-
-	if [ -f "$COMPOSE_PLUGIN_PATH" ] ; then
-		known_plugin_commands+=("compose")
-	fi
+	local plugin_name=""
+	for plugin_path in ${PLUGINS_PATH//,/ }; do
+		plugin_name=$(basename "$plugin_path" | sed 's/ *$//g')
```

Comment:
The global flag in the sed expression does not make sense for a pattern that is nailed to end of line.
```suggestion
		plugin_name=$(basename "$plugin_path" | sed 's/ *$//')
```